components/brave_wallet_ui/components/extension/connected-panel/index.tsx

```diff
@@ -45,8 +46,10 @@ export default class ConnectedPanel extends React.PureComponent<Props> {
   render () {
     const FiatBalance = this.props.selectedAccount.balance * 2000
     const { selectedAccount, isConnected, connectAction, navAction } = this.props
+    const bg = background({ seed: selectedAccount.address })
+    const orb = create({ seed: selectedAccount.address, size: 8, scale: 16 }).toDataURL()
```

Comment:
This seems computationally expensive to run on every re-render. I suggest wrapping in a React Hook, like `useMemo` which would only get called again if `selectedAddress.address` changes.

Reply:
Updated this to be a functional component and changed to useMemo Hook!